### PR TITLE
fix(statusApp): resolving qml errors/warnings pt2.1

### PR DESCRIPTION
**Pull request by alexandraB99**
Relates to https://github.com/status-im/status-desktop/issues/8864 fixes
Closes https://github.com/status-im/status-desktop/issues/9326

### What does the PR do
status app resolving QML errors/warnings

### Affected areas
status app

### CODE REVIEW

**Review by alexjba — APPROVED** [review]
LGTM! Tested and looks good. There is an issue where CommunitySettings cannot be scrolled, but this issue is also reproducible on master.

**Comment by alexandraB99** [comment]
> LGTM! Tested and looks good. There is an issue where CommunitySettings cannot be scrolled, but this issue is also reproducible on master.

this is true, I wasn't sure if I should "fix" this since it's the same on master

**Comment by alexandraB99** [comment]
> LGTM! Tested and looks good. There is an issue where CommunitySettings cannot be scrolled, but this issue is also reproducible on master.

@alexjba fixed due to https://github.com/status-im/status-desktop/issues/9326

File: ui/app/AppLayouts/Wallet/views/CollectiblesView.qml [hunk 1]
```diff
@@ -8,6 +8,8 @@ import StatusQ.Components 0.1
 import shared.panels 1.0
 import utils 1.0
 
+import utils 1.0
```
Comment (alexandraB99) [inline ui/app/AppLayouts/Wallet/views/CollectiblesView.qml:11]: this is needed as `Style` was undefined

**Review by caybro — APPROVED** [review]
LGTM

**Pull request merged**